_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1135,21 +1275,27 @@ Set the variables for the `account-baseline-root` module in this environment in
 inputs = {
   # Fill in your region you want to use (only used for API calls) and the ID of your root AWS account
   aws_region     = "us-east-2"
-  aws_account_id = "111122223333"
+  aws_account_id = "<ROOT_ACCOUNT_ID>"
 
   # Prefix all resources with this name
   name_prefix = "<COMPANY_NAME>-root"
 
-  // If you're running the example against an account that doesn't have AWS Organization created, change the following value to true
+  # If you've already created an AWS Organization in your root account, either run 'import' or set this variable to false
   create_organization = true
 
-  child_accounts = var.child_accounts
+  # On the first deploy, we must disable AWS Config and CloudTrail, as we want to send AWS Config and CloudTrail data
+  # to the logs account, but before we can do that, we have to create and configure the logs account!
```

Comment:
I think it reads more clearly to say
```suggestion
  # On the first deploy, we must disable AWS Config and CloudTrail, which send AWS Config and CloudTrail data
  # to the logs account, which is not yet created. So first we have to create and configure the logs account!
```

Reply:
Hm, I find the original version a bit easier to read / follow. Anyone else wants to chime in with a 3rd opinion to break the tie?

Reply:
Yeah, I don't think either suggestion is great. It should probably say

```suggestion
  # At this point we don't yet have a logs account, so disable AWS Config and CloudTrail, since they depend on the logs account.
```

Reply:
Rephrased to: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/344/commits/19a2b2c443a102f21545caa244e87d18dccd4cf5